usecase/usecase-cordova-android-tests/samples/CordovaPackage/res/test.py

```diff
@@ -126,9 +129,10 @@
     elif BUILD_PARAMETERS.pkgarch == "arm64":
         pack_arch_type = "armv7 --xwalk64bit"
     pack_arch_opt = "-- --gradleArg=-PcdvBuildArch=%s" % pack_arch_type
+    min_sdk_version = "--minSdkVersion=16"
 
 
-os.system("cordova build android %s" % pack_arch_opt)
+os.system("cordova build android %s %s" % (pack_arch_opt, min_sdk_version))
 os.system("cordova run android %s" % pack_arch_opt)
```

Comment:
Update   to `os.system("cordova run android  %s %s" % (pack_arch_opt, min_sdk_version))`